src/vs/workbench/contrib/terminalContrib/links/browser/terminalLocalLinkDetector.ts

```diff
@@ -52,6 +52,8 @@ const fallbackMatchers: RegExp[] = [
 	// C:\foo/bar baz:339: error ...
 	// C:\foo/bar baz:339:12: error ...     [#178584, Clang]
 	/^(?<link>(?<path>.+):(?<line>\d+)(?::(?<col>\d+))?) ?:/,
+	// PowerShell prompt with spaces - PS D:\workspace\abc def>
+	/^PS\s+(?<link>(?<path>[^>]+))>\s*$/,
 	// Cmd prompt
 	/^(?<link>(?<path>.+))>/,
```

Comment:
Let's merge these together with the comment `// PowerShell and cmd prompt`. Also no need to match to the end of the line here, only to the first `>`